addons/configtweaks/config.cpp

```diff
@@ -239,24 +247,426 @@ class CfgWeapons {
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};	
+	class rhs_mag_20Rnd_SCAR_762x51_m80_ball: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_762x51_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+	class rhs_mag_9x19_17: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_9x19_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;	
+	};
+	class rhs_mag_9x18_12_57N181S: rhs_mag_9x19_17
+	{
+		greenmag_basicammo = "greenmag_ammo_9x18_basic_1Rnd";
+	};
+	class rhs_18rnd_9x21mm_7N28: rhs_mag_9x19_17
+	{
+		greenmag_basicammo = "greenmag_ammo_9x21_basic_1Rnd";
+	};
+	
+	// GREF
+	class rhsgref_30rnd_556x45_m21: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_556x45_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};	
+	class rhsgref_30rnd_556x45_vhs2: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_556x45_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+	class rhsgref_30rnd_1143x23_M1911B_SMG: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_45ACP_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+	
+	// USF
+	class rhsusf_100Rnd_556x45_soft_pouch: rhs_mag_30Rnd_556x45_M855A1_Stanag
+	{
+		greenmag_ammo = "greenmag_beltlinked_556x45_basic";
+		greenmag_basicammo = "greenmag_beltlinked_556x45_basic";
+		greenmag_canSpeedload = 0;
+		greenmag_needBelt = 1;
+	};
+	class rhsusf_20Rnd_762x51_m118_special_Mag: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_762x51_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+	class rhsusf_5Rnd_762x51_m118_special_Mag: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_762x51_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};	
+	class rhsusf_10Rnd_762x51_m118_special_Mag: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_762x51_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};	
+	class rhsusf_50Rnd_762x51: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_beltlinked_762x51_basic";
+		greenmag_canSpeedload = 0;
+		greenmag_needBelt = 1;		
+	};	
+	class rhsusf_mag_40Rnd_46x30_AP: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_46x30_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+	class rhsusf_mag_40Rnd_46x30_FMJ: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_46x30_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};		
+	class rhsusf_mag_40Rnd_46x30_JHP: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_46x30_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};	
+	class rhsusf_mag_17Rnd_9x19_FMJ: CA_Magazine
+	{
+		greenmag_basicammo = "greenmag_ammo_9x19_basic_1Rnd";
+		greenmag_canSpeedload = 1;
+		greenmag_needBelt = 0;		
+	};
+};
+
+// Ammo Balance, normalizes 12G slugs across mods, example of what can be done with this
```

Comment:
Worth noting that if a mod _does_ set its own values, the inheritance is overwritten, which is a pain to track down but ye.

Reply:
There aren't so many ammo calibers so it's not too hard to go through, and I found a spreadsheet with values for different mods, that at least makes it easier to know where to look.